src/main/resources/pdf_templates/ds01.html

```diff
@@ -0,0 +1,209 @@
+{{"<?xml version=\"1.0\" encoding=\"UTF-8\"?>" | unescaped}}
+
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml"
+      xml:lang="en">
+
+<head>
+    <meta content="application/xhtml+xml; charset=UTF-8" http-equiv="content-type"/>
+    <title>DS01</title>
+
+    <style type="text/css">
+        body {
```

Comment:
Looking at the PDFs they have a significant amount of padding around the outside, yet i see no reference to padding here. Maybe reconsider your use of the HTML box model for spacing elements.